README.md

```diff
@@ -23,4 +23,117 @@ You are given an array of goods. Render them in a table with the ability to sele
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_goods-selector/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://spencerspe.github.io/react_goods-selector/) and add it to the PR description.
+import React from 'react';
+import 'bulma/css/bulma.css';
+import './App.scss';
+import { render } from 'react-dom';
+import { stringify } from 'querystring';
+
+export const goods = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+type State = {
+  selectedGood: string;
+  selectedStatus: boolean;
+}
+
+export class App extends React.Component<{}, State>  {
+
+state: State = {
+  selectedGood: 'Jam',
+  selectedStatus: false,
+}
+
+resetButtons = () => {
+  const buttons = document.querySelectorAll('button');
+  buttons.forEach(button => button.textContent = '+');
+
+  const tableRows = document.querySelectorAll('tr');
+  tableRows.forEach(row => row.classList.remove('has-background-success-light'))
+
+}
+
+beenSelected = (event: React.MouseEvent, item: string) => {
+  this.resetButtons();
+  const clicked = event.target as HTMLButtonElement;
+  const clickedTr = document.getElementById(`${clicked.id}-0`);
+  clickedTr.classList.add('has-background-success-light');
+  clicked.textContent = '-'
+
+  this.setState({
+    selectedGood: item,
+    selectedStatus: false,
+})
+}
+
+render() {
+
+  return (
+<main className="section container">
+    <h1 className="title">
+    {this.state.selectedStatus ? (
+  <>No goods selected</>
+) : (
+  <>
+    <>{`${this.state.selectedGood} is selected`}</>
+    {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+    <button
+      data-cy="ClearButton"
+      type="button"
+      className='delete ml-3'
+      onClick={() => {
+        this.setState({ selectedStatus: true})
+        this.resetButtons();
+      }}
```

Comment:
Pls move the function inside `onClick` handler to a separate method and use as a callback here